src/main/java/org/primeframework/mvc/action/config/DefaultActionConfigurationBuilder.java

```diff
@@ -136,7 +138,13 @@ public ActionConfiguration build(Class<?> actionClass) {
     Field unknownParametersField = findUnknownParametersField(actionClass);
     Set<String> validContentTypes = findAllowedContentTypes(actionClass);
 
-    return new ActionConfiguration(actionClass, allowKnownParameters, constraintValidationMethods, executeMethods, validationMethods, formPrepareMethods, authorizationMethods, jwtAuthorizationMethods, postValidationMethods, preParameterMethods, postParameterMethods, resultAnnotations, preParameterMembers, preRenderMethodsMap, fileUploadMembers, memberNames, securitySchemes, scopeFields, additionalConfiguration, uri, preValidationMethods, unknownParametersField, validContentTypes);
+    List<String> fallbackActionMessageURIs = Optional.ofNullable(actionClass.getAnnotation(AlternateMessageResources.class))
```

Comment:
This is just my comment because I wanted to prove it to myself.

But this code assumes that we are building an ordered collection from the values we find in `alternates.actions()` and we are using a `List` type. 

So I read the `Stream.toList()` doc and the default impl is an `ArrayList` and the interface actual specified that the result will be in the streams encounter order, and must be equal to this code:

```
Collections.unmodifiableList(new ArrayList<>(Arrays. asList(this.toArray())))
```

So that is all to say, order should be maintained as we expect.